jina/jaml/helper.py

```diff
@@ -87,8 +87,20 @@ def parse_config_source(path: Union[str, TextIO, Dict],
                         allow_dict: bool = True,
                         allow_json: bool = True,
                         *args, **kwargs) -> Tuple[TextIO, Optional[str]]:
-    """ Check if the text or text stream is valid
-
+    """Check if the text or text stream is valid.
+
+    # noqa: DAR401
+    :param path: the multi-kind source of the configs.
```

Comment:
I do not see how to add suggestions now?

Reply:
I merged it after receiving two approvals

Reply:
You can make a new PR